addon/modifiers/in-viewport.js

```diff
@@ -39,14 +39,14 @@ export default class InViewportModifier extends Modifier {
   @action
   onEnter() {
```

Comment:
Do we get the IntersectionEntry here as an argument?  I believe we do.  In that case, we could just open up this function's arguments to that specific argument.  What do you think?